instrumentation/opentelemetry-instrumentation-dbapi/src/opentelemetry/instrumentation/dbapi/__init__.py

```diff
@@ -350,11 +350,25 @@ def calculate_commenter_data(self) -> dict[str, Any]:
         }
 
         if self.database_system == "postgresql":
-            if hasattr(self.connect_module, "__libpq_version__"):
-                libpq_version = self.connect_module.__libpq_version__
-            else:
-                libpq_version = self.connect_module.pq.__build_version__
-            commenter_data.update({"libpq_version": libpq_version})
+            libpq_version = None
+            # psycopg
+            if hasattr(self.connect_module, "pq"):
+                try:
+                    libpq_version = self.connect_module.pq.version()
+                except Exception:  # pylint: disable=broad-exception-caught
+                    pass
```

Comment:
nit would this work?
```suggestion
            try:
                libpq_version = self.connect_module.pq.version()
            except Exception:  # pylint: disable=broad-exception-caught
                pass
```